hide/client/hide_client.py

```diff
@@ -120,9 +120,17 @@ def update_file(
             raise HideClientError(response.text)
         return model.File.model_validate(response.json())
 
-    def delete_file(self, project_id: str, path: str) -> bool:
+    def delete_file(
+        self, project_id: str, file: str | model.File | model.FileInfo
```

Comment:
I think we should type filepath, `file: model.FilePath | model.File | model.FileInfor`. I feel like file as a string can be very confusing.

Reply:
done